Makefile

```diff
@@ -937,12 +945,22 @@ set-manifest-image:
 clean: ## Remove all generated files
 	$(MAKE) clean-bin
 	$(MAKE) clean-book
+	$(MAKE) clean-tilt
+	$(MAKE) clean-charts
 
 .PHONY: clean-bin
 clean-bin: ## Remove all generated binaries
 	rm -rf $(BIN_DIR)
 	rm -rf $(TOOLS_BIN_DIR)
 
+.PHONY: clean-tilt
+clean-tilt: ## Remove all files in .tiltbuild
+	rm -rf ./tiltbuild
```

Comment:
I am good with adding this target but just FYI `tilt-prepare` takes care of updating the files in `.tiltbuild` if it observes any changes. Dropping this comment here incase this target was just added as part of the "pull the correct chart" problem.

Reply:
Yeah I added it since @sbueringer mentioned a while back we didn't have any way to clean up unused files in `.tiltbuild`. For example when we change the output filename it won't remove the old file.

Reply:
This doesn't remove the .tiltbuild stuff in e.g. docker, controlplane, bootstrap, right? I guess we should remove `./**/.tiltbuild` ?

This also won't clean artifacts in provider repos, so it could be slightly confusing, but those are just binaries I think.

Reply:
You are right, the `.tiltbuild` folders under providers only contain the manager binaries for now. I think we should be fine with just cleaning up `.tiltbuild` in the CAPI root.

Reply:
Let's delete all of them. Only deleting one is a bit inconsequential. I'm fine with just listing the 4 rm -rf calls here. After all they are pretty stable in core CAPI